cdk/lib/db.ts

```diff
@@ -234,57 +245,90 @@ export class CacclDocDb extends CacclDbBase {
     this.metrics = {
       ReadIOPS: [this.makeDocDbMetric('ReadIOPS')],
       WriteIOPS: [this.makeDocDbMetric('WriteIOPS')],
-      CPUUtilization: [this.makeDocDbMetric('CPUUtilization', { unit: Unit.PERCENT })],
+      CPUUtilization: [
+        this.makeDocDbMetric('CPUUtilization', {
+          unit: cloudwatch.Unit.PERCENT,
+        }),
+      ],
       FreeableMemory: [this.makeDocDbMetric('FreeableMemory')],
-      BufferCacheHitRatio: [this.makeDocDbMetric('BufferCacheHitRatio', { unit: Unit.PERCENT })],
+      BufferCacheHitRatio: [
+        this.makeDocDbMetric('BufferCacheHitRatio', {
+          unit: cloudwatch.Unit.PERCENT,
+        }),
+      ],
       DatabaseConnections: [this.makeDocDbMetric('DatabaseConnections')],
       DiskQueueDepth: [this.makeDocDbMetric('DiskQueueDepth')],
-      ReadLatency: [this.makeDocDbMetric('ReadLatency', { unit: Unit.MILLISECONDS })],
-      WriteLatency: [this.makeDocDbMetric('WriteLatency', { unit: Unit.MILLISECONDS })],
-      DatabaseCursorsTimedOut: [this.makeDocDbMetric('DatabaseCursorsTimedOut', { statistic: 'sum' })],
+      ReadLatency: [
+        this.makeDocDbMetric('ReadLatency', {
+          unit: cloudwatch.Unit.MILLISECONDS,
+        }),
+      ],
+      WriteLatency: [
+        this.makeDocDbMetric('WriteLatency', {
+          unit: cloudwatch.Unit.MILLISECONDS,
+        }),
+      ],
+      DatabaseCursorsTimedOut: [
+        this.makeDocDbMetric('DatabaseCursorsTimedOut', { statistic: 'sum' }),
+      ],
       Transactions: [this.makeDocDbMetric('TransactionsOpen')],
       Queries: [this.makeDocDbMetric('OpcountersQuery')],
     };
 
     this.alarms = [
-      new Alarm(this, 'CPUUtilizationAlarm', {
-        metric: this.metrics.CPUUtilization[0].with({ period: Duration.minutes(5) }),
+      new cloudwatch.Alarm(this, 'CPUUtilizationAlarm', {
+        metric: this.metrics.CPUUtilization[0].with({
+          period: Duration.minutes(5),
+        }),
         threshold: 50,
         evaluationPeriods: 3,
-        alarmDescription: `${Stack.of(this).stackName} docdb cpu utilization alarm`,
+        alarmDescription: `${
+          Stack.of(this).stackName
+        } docdb cpu utilization alarm`,
       }),
-      new Alarm(this, 'BufferCacheHitRatioAlarm', {
+      new cloudwatch.Alarm(this, 'BufferCacheHitRatioAlarm', {
         metric: this.metrics.BufferCacheHitRatio[0],
         threshold: 90,
         evaluationPeriods: 3,
-        comparisonOperator: ComparisonOperator.LESS_THAN_OR_EQUAL_TO_THRESHOLD,
-        alarmDescription: `${Stack.of(this).stackName} docdb buffer cache hit ratio alarm`,
+        comparisonOperator:
+          cloudwatch.ComparisonOperator.LESS_THAN_OR_EQUAL_TO_THRESHOLD,
+        alarmDescription: `${
+          Stack.of(this).stackName
```

Comment:
Yeah I do think that strings like this might look better if not broken onto multiple lines.